full.rollbar/project.clj

```diff
@@ -0,0 +1,9 @@
+(defproject fullcontact/full.rollbar "0.1.0-SNAPSHOT"
```

Comment:
project.clj should look like this: https://github.com/fullcontact/full.monty/blob/master/full.http/project.clj

(just change the description and dependencies) 

That's also why travis build is failing.